README.md

```diff
@@ -139,17 +139,21 @@ You'll need a C3 or C3X and a Toyota A harness for current supported vehicles.
 
 ### Step 1. Power up C3X, connect to Wi-Fi, and install `commaai/master-ci`
 
-1-1. Start with the official installation guide: https://comma.ai/setup/comma-3x
+1-1. Install the hardware by following the official Installation Guide: https://comma.ai/setup/comma-3x
 
-1-2. Always use the right-angled USB-C cable that came with the Comma 3X. Don't use your own USB cable. A USB-C 3.1 Gen 2 is required.
+1-2. Turn on your Comma 3X by plugging in the right-angled OBD-C cable to the harness.
 
-1-3. Also connect the OBD2 connector to make sure the Comma stays powered on while turning the car on and off.
+> [!WARNING]
+> * For connecting Comma to the car harness, always use the right-angled OBD-C cable that came with the Comma 3X. The hardness sends 12V instead of the usual 5V, so your own USB cable will not work.
```

Comment:
case thing as mentioned. Also harness not hardness. 

The voltage is not related things working or not. However, maybe it should be noted that it is *not* USB and that plugging things in that aren't C3s may cause damage.

Reply:
Is it the speed that's needed?

Reply:
It's just all pins that's needed.